### ISSUE TITLE
Signature: Invalid SFV format for Signature-Input field

### ISSUE THREAD

**Issue by Jxck** [httpwg/http-extensions#1390]
https://tools.ietf.org/html/draft-ietf-httpbis-message-signatures-01#section-4.3

```http
 Signature-Input: sig1=(*request-target, *created, host, date,
       cache-control, x-empty-header, x-example); keyId="test-key-a";
       alg=hs2019; created=1402170695; expires=1402170995
```
[result](https://jxck.github.io/structured-field-values/demo.html#c2lnMT0oKnJlcXVlc3QtdGFyZ2V0LCAqY3JlYXRlZCwgaG9zdCwgZGF0ZSxjYWNoZS1jb250cm9sLCB4LWVtcHR5LWhlYWRlciwgeC1leGFtcGxlKTsga2V5SWQ9InRlc3Qta2V5LWEiO2FsZz1oczIwMTk7IGNyZWF0ZWQ9MTQwMjE3MDY5NTsgZXhwaXJlcz0xNDAyMTcwOTk1)

InnerList is not separated by `,` but ` `.
Key for Dict should not contain uppercase so `keyId` is invalid, use `key_id`, `keyid` can be use.

```http
sig1=(*request-target *created host date cache-control x-empty-header x-example); key_id="test-key-a"; alg=hs2019; created=1402170695; expires=1402170995
```

[result](https://jxck.github.io/structured-field-values/demo.html#c2lnMT0oKnJlcXVlc3QtdGFyZ2V0ICpjcmVhdGVkIGhvc3QgZGF0ZSBjYWNoZS1jb250cm9sIHgtZW1wdHktaGVhZGVyIHgtZXhhbXBsZSk7IGtleV9pZD0idGVzdC1rZXktYSI7IGFsZz1oczIwMTk7IGNyZWF0ZWQ9MTQwMjE3MDY5NTsgZXhwaXJlcz0xNDAyMTcwOTk1)

**Comment by jricher** [comment]
Should be fixed in #1411.

**Comment by jricher** [comment]
Fixed in -03 and fixed harder in -04
